src/Symfony/Component/Messenger/Tests/Transport/RedisExt/ConnectionTest.php

```diff
@@ -86,20 +86,6 @@ public function testFromDsnWithQueryOptions()
         );
     }
 
-    public function testKeepGettingPendingMessages()
```

Comment:
This test should still work the same way as before.

Reply:
I've readded the method. The diff is just damaged because I moved it down after `testAuth` which does make more sense IMO.

Reply:
I see